bgpd/bgp_nb_config.c

```diff
@@ -6277,6 +6277,9 @@ static struct peer *bgp_peer_group_peer_lookup(struct bgp *bgp,
 	struct peer_group *group = NULL;
 
 	group = peer_group_lookup(bgp, peer_str);
+	if (!group)
+		return peer;
+
```

Comment:
Since you are in here, can you just `return group->conf;` on the next line as well as remove line 6276 and I'll withdraw my PR https://github.com/FRRouting/frr/pull/7470

Reply:
👍

Reply:
I guess we should return NULL instead of group->conf because the group can be NULL.